flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/xa/JdbcXaSinkFunctionState.java

```diff
@@ -31,7 +31,7 @@
 /** Thread-safe (assuming immutable {@link Xid} implementation). */
 @ThreadSafe
 @Internal
-class JdbcXaSinkFunctionState {
+public class JdbcXaSinkFunctionState {
```

Comment:
do wee need change modifier here?
It seems with package private it works ok or did i miss anything?

Reply:
it was from first attempt to reuse this class, but was difficult..
reverted